api-report/tree.api.md

```diff
@@ -443,18 +435,12 @@ export interface FieldChange {
     revision?: RevisionTag;
 }
 
-// @alpha (undocumented)
-export interface FieldChangeEncoder<TChangeset> {
-    decodeJson(formatVersion: number, change: JsonCompatibleReadOnly, decodeChild: NodeChangeDecoder): TChangeset;
-    encodeForJson(formatVersion: number, change: TChangeset, encodeChild: NodeChangeEncoder): JsonCompatibleReadOnly;
-}
-
 // @alpha
 export interface FieldChangeHandler<TChangeset, TEditor extends FieldEditor<TChangeset> = FieldEditor<TChangeset>> {
     // (undocumented)
-    editor: TEditor;
+    codecsFactory: (childCodec: IJsonCodec<NodeChangeset>) => ICodecFamily<TChangeset>;
```

Comment:
Not for this PR, but I'd love if we could manage to type erase the encoder/codec stuff out of the package public API.